raiden/tests/utils/eth_node.py

```diff
@@ -168,37 +169,35 @@ def parity_to_cmd(
     return cmd
 
 
-def geth_create_account(datadir: str, privkey: bytes):
-    """
-    Create an account in `datadir` -- since we're not interested
-    in the rewards, we don't care about the created address.
+def geth_keystore(datadir: str) -> str:
+    return os.path.join(datadir, "keystore")
 
-    Args:
-        datadir: the datadir in which the account is created
-        privkey: the private key for the account
-    """
-    keyfile_path = os.path.join(datadir, "keyfile")
-    with open(keyfile_path, "wb") as handler:
-        handler.write(remove_0x_prefix(encode_hex(privkey)).encode())
-
-    create = subprocess.Popen(
-        ["geth", "--datadir", datadir, "account", "import", keyfile_path],
-        stdin=subprocess.PIPE,
-        universal_newlines=True,
-        stdout=subprocess.DEVNULL,
-        stderr=subprocess.DEVNULL,
-    )
 
-    create.stdin.write(DEFAULT_PASSPHRASE + os.linesep)
-    time.sleep(0.1)
-    create.stdin.write(DEFAULT_PASSPHRASE + os.linesep)
-    create.communicate()
-    assert create.returncode == 0
+def geth_keyfile(datadir: str, address: Address) -> str:
+    keystore = geth_keystore(datadir)
+    os.makedirs(keystore, exist_ok=True)
+
+    address = remove_0x_prefix(to_normalized_address(address))
+    broken_iso_8601 = datetime.now().isoformat().replace(":", "-")
+    account = f"UTC--{broken_iso_8601}000Z--{address}"
```

Comment:
The default resolution from the stdlib `datetime` is not enough, instead of adding another dependency just for this I decided to pad the value with the three zeros.